core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1542,6 +1542,9 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean, dynamicConfigO
   /** ********* Feature configuration ***********/
   def isFeatureVersioningEnabled = interBrokerProtocolVersion >= KAFKA_2_7_IV0
 
+  /** ********* Redirection configuration ***********/
+  def redirectionEnabled = interBrokerProtocolVersion >= KAFKA_2_7_IV1
```

Comment:
I'm still trying to decide how to make sure we could turn off the redirection in 2.7. Having a separate IBP for 3.0 may not work. @cmccabe